linux_os/guide/services/sssd/sssd_run_as_sssd_user/rule.yml

```diff
@@ -0,0 +1,28 @@
+documentation_complete: true
+
+prodtype: rhel8,fedora,ol8
+
+title: 'Configure SSSD to run as user sssd'
+
+description: |-
+    SSSD processes should be configured to run as user sssd, not root.
+
+rationale: |-
+    To minimize privileges of SSSD processes, they are configured to
+    run as non-root user.
+
+severity: medium
+
+identifiers:
+    cce@rhel8: 12345-5
+
+references:
+    ospp: FIA_AFL.1-FIXME-and-identifiers
```

Comment:
Regarding OSPP, I could not identify to which requirement this rule is related.
Closest thing was an Objective for the Operational Environment for the user,([OE.PROPER_USER](https://www.niap-ccevs.org/MMO/PP/-442-/#SecurityObjectivesTOEorEnvironment)), but it is not related to service users, though.

But I found requirements that support this rule, like:

- [FPT_ACF_EXT.1.1](https://www.niap-ccevs.org/MMO/PP/-442-/#FPT_ACF_EXT.1.1)
- [FPT_ACF_EXT.1.1](https://www.niap-ccevs.org/MMO/PP/-442-/#FDP_ACF_EXT.1.1)

@stevegrubb Can you give us a hint?